reader/components/viewer/styles.module.css

```diff
@@ -0,0 +1,20 @@
+.pdfViewer .page {
+  border: none;
+  margin-top: 15px;
+}
+
+.pdfViewer .page:last-of-type {
+  margin-bottom: 15px;
+}
+
+.pdf-metadata {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 1rem;
+  margin: 0 auto;
+}
+
+.textLayer {
+  // has to be higher than .context-menu-layer
```

Comment:
It works because any string that is not recognised by CSS processor is ignored. However, is not proper comment, maybe leftover after SASS.

Reply:
Going to be solved in following PR.